### PR TITLE
Added create adt variables

### PR DESCRIPTION
**Pull request by johnfolly**
- [x] I added create adt variables
- [x] I updated the documentation
- [x] I tested a run with terraform cloud

### CODE REVIEW

**Comment by jpla-csm** [comment]
LGTM, except for commit name, should be create adt variable instead of adx.

**Review by jpla-csm — APPROVED** [review]
LGTM, except for commit name, should be create adt variable instead of adx.

**Comment by johnfolly** [comment]
I fixed it and changed to create adt

File: azure/new-workspace/terraform.tfvars [hunk 1]
```diff
@@ -11,4 +11,4 @@ subscription_id = ""
 resource_group = ""
 organization_id = ""
 workspace_key = ""
-
+create_adt_instance = "false"
```
Comment (enocquet) [inline azure/new-workspace/terraform.tfvars]: let it like : create_adt_instance = ""

**Review by enocquet — CHANGES_REQUESTED** [review]

**Review by enocquet — APPROVED** [review]

**Pull request merged**